src/lib/dataDog.ts

```diff
@@ -0,0 +1,57 @@
+declare global {
+  interface Window {
+    DD_RUM: {
+      setUser: (user: Record<string, unknown>) => void;
+      clearUser: () => void;
+    };
+  }
+}
+
+export const isDataDogConfigured = (): boolean => {
+  if (typeof window === 'undefined') return false;
```

Comment:
I think we should adopt the `curly: ['error', 'all']` lint rule. I'm a huge fan of always having curly braces around if blocks, even if they are one liners. Feel free to discuss. cc @dr-bizz

Reply:
I'm used to the curly braces being everywhere as well, and it always throws me off a little bit when they're not there.

Reply:
#945

Reply:
I'm good with adding it